packages/blocks/README.md

```diff
@@ -193,6 +193,6 @@ OptionObject(dynamicText, 'value', undefined, {
 });
 ```
 
-Transform function signature: `<T>(limit: number, value: T) => T`. Be sure to return an value that is the same type the `value` passed - text element, string, option object array, etc. depending on the field.
+Transform function signature: `<T>(limit: number, value: T) => T`. Be sure to retur an value that is the same type the `value` passed - text element, string, option object array, etc. depending on the field.
```

Comment:
`...return a value that is the same type as the...`

Reply:
ouch, not sure how I messed up this sentence that badly.